tests/app/UnitTests/GitCommands.Tests/ExecutableTests.cs

```diff
@@ -55,7 +55,7 @@ public async Task Process_shall_be_killed_on_cancellation()
             process.Dispose();
 
             TimeSpan durationWaitTimeout = DateTime.Now - startedAt;
-            durationWaitTimeout.Should().BeGreaterThan(1.5 * cancelDelay).And.BeLessThan(2.5 * cancelDelay);
+            durationWaitTimeout.Should().BeGreaterThan(1.5 * cancelDelay).And.BeLessThan(4 * cancelDelay);
```

Comment:
Is 4 enough, why not 10 or so?

Reply:
> Is 4 enough, why not 10 or so?

Most of the time, the testcase passed within the original 2.5 seconds. Worst case was ~ 3.5 seconds.
The process shall be killed in reasonable time. I want to be notified if this does not work well.